src/common/keyboard.js

```diff
@@ -92,7 +92,7 @@ if (!('key' in window.KeyboardEvent.prototype)) {
   ( window.KeyboardEvent.prototype
   , 'key'
   , { get: getKey
-    /*::, value: void(0)*/
+    , value: void(0)
```

Comment:
@tschneidereit I think problem is actually in `Object.defineProperty` API being non typing friendly. It means that descriptor type would have to be:

```js
type PropertyDescriptor <x> = 
  | { value: x }
  | { get: () => x }
  | { set: (x) => any }
  | { get: () => x, set: () => x }
```

And such union types are difficult to support. At least flow does not support them. Alternative would be to define descriptor type as:

```js
type PropertyDescriptor <x> = {
  value?: x,
  get?: () => x,
  set?: (x) => any
}
```

In which case flow won't really be able to tell you if you're making mistake and provide no properties at all or maybe `value` and `set`. 

My guess is they resorted to just making all of the properties mandatory.